.github/workflows/ci.yml

```
@@ -563,8 +585,6 @@ jobs:
     steps:
       - name: Checkout sources
         uses: actions/checkout@v4
-      - uses: Swatinem/rust-cache@v2
-        if: github.event_name != 'release' && github.event_name != 'workflow_dispatch'
       - name: Run NGINX
```

removing this was a mistake as noted in https://github.com/maplibre/martin/pull/1902/files#r2185269914

```suggestion
      - uses: Swatinem/rust-cache@v2
        if: github.event_name != 'release' && github.event_name != 'workflow_dispatch'
      - name: Run NGINX
```